src/Acuminator/Acuminator.Utilities/Common/Constants.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
```

Comment:
@saloponov the Common folder is not a very good place for constant file - it was designed to store common helpers for common .net classes and interfaces.
The good place will be imho a separate folder Constants in Roslyn folder or at root level. Or at least move file to the "Roslyn" folder